ui/tests/acceptance/deployments-list-test.ts

```diff
@@ -7,29 +7,55 @@ import { setupMirage } from 'ember-cli-mirage/test-support';
 import { setupSession } from 'waypoint/tests/helpers/login';
 
 const url = '/default/microchip/app/wp-bandwidth/deployments';
+const emptyStateUrl = '/default/microchip/app/wp-bandwidth/deployment';
+const redirectUrl = '/default/microchip/app/wp-bandwidth/deployment/seq/'; // concat with length of deployments array in test
 
 const page = create({
   visit: visitable(url),
   list: collection('[data-test-deployment-list] li'),
-  deploymentLinks: collection('[data-test-external-deployment-button]'),
   destroyedBadges: collection('[data-test-destroyed-badge]'),
   showDestroyed: clickable('[data-test-display-destroyed-button]'),
+  linkList: collection('[data-test-deployment-list-item]'),
 });
 
 module('Acceptance | deployments list', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
   setupSession(hooks);
 
-  test('visiting deployments page', async function (assert) {
+  test('visiting deployments page redirects to latest', async function (assert) {
     let project = this.server.create('project', { name: 'microchip' });
     let application = this.server.create('application', { name: 'wp-bandwidth', project });
     this.server.createList('deployment', 3, 'random', { application });
 
     await page.visit();
 
     assert.equal(page.list.length, 3);
-    assert.equal(currentURL(), url);
+    assert.equal(currentURL(), redirectUrl + 3);
```

Comment:
I’d be tempted to wrap some quotes around that 3 just to make it clear it’s an identifier, not a magic number.

```suggestion
    assert.equal(currentURL(), redirectUrl + '3');
```